source/getting_started/on-vm.rst

```diff
@@ -95,121 +53,49 @@ caas-flashfiles-eng.<user>.zip
-
-#. Run the :command:`lsusb` command. Use the output to identify the vendor
-   ID and product ID of the USB flash drive.
-   :
+.. note::
+        This section is for flashing *flashfiles* package, no action is required if the image caas.img is used.
 
-    .. code-block:: bash
-
-        $ lsusb
-        ...
-        Bus 002 Device 002: ID 0781:5591 SanDisk Corp.
-        ...
-
-    In the example above, *0781* and *5591* are the vendor ID and
-    product ID, respectively, of the target USB flash drive.
-
-#. Identify the directory on which you mounted the USB flash drive. If the
-   flash drive is not mounted, mount the USB flash drive to a temporary
-   directory. Unzip the content of the CaaS *flashfile* package to the flash
-   drive. Unmount the USB flash drive after finishing:
-
-    .. code-block:: bash
-
-        $ sudo mount /dev/sdc /mnt   ## if the USB disk is not auto-mounted
-        $ sudo unzip caas-flashfiles-eng.<user>.zip -d /mnt
-        $ umount /dev/sdc
-
-    .. note::
-        The previous example assumes you assigned :file:`/dev/sdc` to the
-        USB flash drive. You might need to replace the drive names with the
-        actual device node observed from the :command:`lsblk` command.
-
-#. Download the helper script :file:`start_flash_usb.sh`. Edit the script
-   with the vendor ID and product ID for your USB flash drive obtained from
-   the previous step:
-
-    .. code-block:: bash
-
-        $ wget https://raw.githubusercontent.com/projectceladon/device-androidia-mixins/master/groups/device-specific/caas/start_flash_usb.sh  -O ~/caas/start_flash_usb.sh
-        $ chmod +x ~/caas/start_flash_usb.sh
+Follow the instructions below to create and set up CaaS partitions on
+a *qcow2* formatted virtual disk.
 
-    .. code-block:: none
-
-        #!/bin/bash
-
-        qemu-system-x86_64 \
-        ...
-        -device usb-host,bus=xhci.0,vendorid=<your-usb-vendorid>,productid=<your-usb-productid> \
-        ...
-        -bios ./OVMF.fd \
-
-#. After launching the :file:`start_flash_usb.sh` script, a QEMU window
-   pops up. Click the QEMU window, press the :kbd:`F2` key, and select the
-   :menuselection:`Boot Manager` option to enter the boot-manager menu:
+#. Run the helper script :file:`start_flash_usb.sh`.
 
     .. code-block:: bash
 
         $ cd ~/caas
-        $ sudo ./start_flash_usb.sh
-
-    .. figure:: images/qemu-bios.png
-        :align: center
-
-#. Select the :menuselection:`EFI Internal Shell` entry to run the built-in
-   UEFI Shell. The virtual UEFI firmware will detect the USB flash drive and
-   start flashing the CaaS partition images to the :file:`android.qcow2`
-   virtual disk.
+        $ sudo ./start_flash_usb.sh caas-flashfiles-eng.<user>.zip
 
-    .. figure:: images/qemu-bios-bootmanager.png
-        :align: center
+#. A QEMU window will be poped up on the running of
+   :file:`start_flash_usb.sh`.
+   It will drop to the built-in UEFI Shell and starting flash.
```

Comment:
By running the :file:`start_flash_usb.sh` script, a QEMU will be popped up, it will drop to the built-in UEFI Shell and start flashing the partitions to the virtual disk image.